### PR TITLE
updated template

### PR DESCRIPTION
**Pull request by mmn0o7**
Added new variables for autoscaling control instead of min max nodes

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -30,24 +31,58 @@ variable "resource_labels" {
 // ----------------------------------------------------------------------------
 // cluster configuration
 // ----------------------------------------------------------------------------
+variable "master_authorized_networks" {
```
Comment (tomhobson) [inline variables.tf:33]: can we document this field in the readme?

**Review by tomhobson — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (mmn0o7) [reply variables.tf:33]: added a readme for this

**Comment by tgelpi** [comment]
I was trying to build a new GKE using this config and having issues with quotas. Quotas SSD_TOTAL_GB  and IN_USE_ADDRESSES had to be doubled to 1000GB and 16IPs respectively.  I'm using the default 3/5 min/max settings. Is this quota increase a new requirement?

File: variables.tf [hunk 2]
```diff
@@ -4,6 +4,7 @@
 variable "gcp_project" {
   description = "The name of the GCP project to use"
   type        = string
+  default = "filedriver-291811"
```
Comment (tomhobson) [inline variables.tf]: can we remove this default

**Review by tomhobson — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 2 above)
Comment (thenashef) [reply variables.tf]: Good catch, removed

**Review by tomhobson — APPROVED** [review]

**Pull request merged**